### PR TITLE
Improve the iframe resize example page.

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: test/manual/amp-iframe-resize.amp.html [hunk 1]
```diff
@@ -5,59 +5,53 @@
...
+  <div style="height: 120vh"></div>
+
+  <div id="panel">
+    <label for="width">width</label><input id="width" type="number" value="200">
+    <label for="height">height</label><input id="height" type="nubmer" value="200">
+    <button onclick="requestResize()">Resize</button>
+  </div>
+
+  <script>
+    function requestResize() {
```
Comment (cvializ) [inline test/manual/amp-iframe-resize.amp.html]: Are these manual test pages supposed to be valid AMPs? I believe this script tag will make the page fail validation. Is that true?

File: test/manual/amp-iframe-resize.amp.html [hunk 2]
```diff
@@ -5,59 +5,53 @@
...
+      padding: 5px;
     }
   </style>
   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
-  <script async src="../../dist/amp.js"></script>
 </head>
 <body>
-  The iframes in the initial viewport should not be resized automatically after 2 seconds.
+  <h1>Scroll down to see the iframe</h1>
+  <div style="height: 120vh"></div>
```
Comment (cvializ) [inline test/manual/amp-iframe-resize.amp.html]: Since you write this style in two different places it should be a CSS class.

Comment (lannka) [inline test/manual/amp-iframe-resize.amp.html]: done

File: test/manual/amp-iframe-resize.amp.html (hunk 1 above)
Comment (lannka) [inline test/manual/amp-iframe-resize.amp.html]: it will fail validation. but should be fine for such a manual test page. It was invalid anyway.

Comment (cvializ) [inline test/manual/amp-iframe-resize.amp.html]: Sounds good to me

**Pull request merged**